parsers/corelight-sensor.yaml

```diff
@@ -224,101 +657,86 @@ script: |
 
   // File Fields
-  | file.size := rename(Vendor.total_bytes)
+  | file.size := Vendor.total_bytes
 
   // Hash Fields
-  | file.hash.md5 := rename(Vendor.md5)
-  | file.hash.sha256 := rename(Vendor.sha256)
-  | lower(file.hash.sha256, as="file.hash.sha256")
-  | lower(file.hash.md5, as="file.hash.md5")
+  | file.hash.md5 := lower(Vendor.md5)
+  | file.hash.sha256 := lower(Vendor.sha256)
 
   // HTTP Fields
-  | http.request.method := rename(Vendor.method)
-  | url.path := rename(Vendor.uri)
-  | url.domain := rename(Vendor.host)
-  | http.response.status_code := rename(Vendor.status_code)
+  | http.request.method := Vendor.method
+  | url.path := Vendor.uri
+  | url.domain := Vendor.host
+  | http.response.status_code := Vendor.status_code
   | case {
-  // Manually parse out mime response type
-      _path = "dns" AND Vendor.resp_mime_types[1] = * | @rawstring = /resp_mime_types":\[(?<http.response.mime_type>.*?)\]/;
-      _path = "dns" AND Vendor.resp_mime_types[0] = * | http.response.mime_type := rename(Vendor.resp_mime_types[0]);
+      // Manually parse out mime response type
+      _path = "http" AND Vendor.resp_mime_types[1] = * | @rawstring = /resp_mime_types":\[(?<http.response.mime_type>.*?)\]/;
+      _path = "http" AND Vendor.resp_mime_types[0] = * | http.response.mime_type := Vendor.resp_mime_types[0];
       *;
   }
   | case {
-  // Manually parse out mime request type
-      _path = "dns" AND Vendor.orig_mime_types[1] = * | @rawstring = /orig_mime_types":\[(?<http.request.mime_type>.*?)\]/;
-      _path = "dns" AND Vendor.orig_mime_types[0] = * | http.request.mime_type := rename(Vendor.orig_mime_types[0]);
+      // Manually parse out mime request type
+      _path = "http" AND Vendor.orig_mime_types[1] = * | @rawstring = /orig_mime_types":\[(?<http.request.mime_type>.*?)\]/;
+      _path = "http" AND Vendor.orig_mime_types[0] = * | http.request.mime_type := Vendor.orig_mime_types[0];
       *;
   }
 
-
   // DNS Fields
   | case {
-  // Manually parse out dns.answers
-  _path = "dns" | @rawstring = /answers":\[(?<dns.answers>.*?)\]/;
-  *;
+      // Manually parse out dns.answers
+      _path = "dns" | @rawstring = /answers":\[(?<dns.answers>.*?)\]/;
+      *;
   }
-  | dns.question.type := rename(Vendor.qtype_name)
-  | dns.question.name := rename(Vendor.query)
+  | dns.question.type := Vendor.qtype_name
+  | dns.question.name := Vendor.query
   | splitString(field="Vendor.answers", by=",", as="dns.answers")
-  | dns.response_code := rename(Vendor.rcode_name)
+  | dns.response_code := Vendor.rcode_name
 
   // Email Fields
-  | email.sender.address := rename(Vendor.mailfrom)
-  | lower(email.sender.address, as="email.sender.address")
-  | email.subject := rename(Vendor.subject)
+  | email.sender.address := lower(Vendor.mailfrom)
+  | email.subject := Vendor.subject
   | case {
       // Manual parsing for email.to.address
       _path = "smtp" | @rawstring = /rcptto":\[(?<email.to.address>.*?)\]/;
```

Comment:
from CRWD
- Can you pls include sample for this case?
ECS expects the `email.to.addres`s to be and if you just extracting it from the rawstring it likely needs more processing to convert it into an array